CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Change Log
 ==========
 
+[Version 0.5](https://github.com/novoda/gradle-static-analysis-plugin/releases/tag/v0.5)
```

Comment:
This link doesn't work yet. The `publishRelease` triggered by the CI will create the tag.